src/constants/attributes.ts

```diff
@@ -9,6 +9,7 @@ export const KEY_EMB_SESSION_REASON_ENDED = 'emb.session_end_type';
 export const KEY_EMB_SESSION_REASON_STARTED = 'emb.session_start_type';
 export const KEY_EMB_JS_EXCEPTION_STACKTRACE = 'emb.stacktrace.js';
 export const KEY_EMB_EXCEPTION_HANDLING = 'emb.exception_handling';
+export const KEY_EMB_EXCEPTION_CAUSE = 'emb.exception_cause';
```

Comment:
This looks good, I should add a debug field for this, I will create a ticket to do it